shared/actions/settings.tsx

```diff
@@ -287,9 +287,9 @@ function* refreshNotifications() {
         ],
       },
       {
-        description: 'Disable sending/receiving typing notifications',
+        description: 'Display typing notifications',
         name: 'disabletyping',
-        subscribed: !!chatGlobalSettings.settings[`${ChatTypes.GlobalAppNotificationSetting.disabletyping}`],
+        subscribed: !!!chatGlobalSettings.settings[`${ChatTypes.GlobalAppNotificationSetting.disabletyping}`],
```

Comment:
I think this can just be `!`, not `!!!`

Reply:
oops forgot to push taht

Reply:
but what if it's, like, really really exciting